sources/api/ecs-settings-applier/src/ecs.rs

```diff
@@ -82,6 +82,11 @@ struct ECSConfig {
     image_cleanup_delete_per_cycle: Option<i64>,
 
     image_cleanup_disabled: bool,
+
```

Comment:
It seems like ECS Agent had enough changes to possibly be worth splitting off into a separate review. They are here now but it could help make review easier when there is already a lot to review to take the more involved ones and separate them.

Reply:
Will keep in mind to have separate PR for the packages having so many changes.